frontend/src/components/ImplementationReportView/DialectCompliance.tsx

```diff
@@ -42,7 +42,18 @@ const DialectCompliance: React.FC<{
               .map(([dialectName, result], index) => {
                 return (
                   <tr key={index}>
-                    <td>{Dialect.forPath(dialectName).uri}</td>
+                    <td>
+                      {Dialect.forPath(dialectName).uri}{" "}
+                      <img
+                        width={"auto"}
+                        height={20}
+                        src={`https://img.shields.io/endpoint?url=https%3A%2F%2Fbowtie-json-schema.github.io%2Fbowtie%2Fbadges%2F${encodeURIComponent(
+                          implementation.language,
+                        )}-${implementation.name}%2Fcompliance%2FDraft_${
+                          Dialect.forPath(dialectName).prettyName.split(" ")[1]
```

Comment:
Got to say that depending your logic on `prettyName` which, as the name suggests, is just for presentation purposes is a bad idea. All the dialects data can be found in file `dialects.json` - I'd say add appropriate value to each dialect, adjust `Dialect.ts`, and then use this value here. NGL this path for badges should be just different (probably include `dialect.shortName`)

Reply:
Added `badgeName`  to each dialects. in `dialects.json` and adjusted Dialect.ts. 
`dialects.json` changes example-

```json
  {
    "$schema": "tag:bowtie.report,2024:models:dialect",
    "uri": "https://json-schema.org/draft/2020-12/schema",
    "shortName": "draft2020-12",
    "prettyName": "Draft 2020-12",
    "badgeName": "Draft_2020-12",
    "firstPublicationDate": "2020-01-28",
    "aliases": ["2020", "202012", "2020-12", "draft202012"]
  }

Reply:
I haven't looked closely but I think you've only changed this on the frontend -- that's going to break, since you haven't changed the path itself in the backend which generates the file, so you'll be looking in the wrong place.
Harrel's totally right that this is very wrong to use `prettyName`, so I agree 100% we should change it, but you have to change it in both places, and if/once you do I think you won't even need this new field, you can just make the badge name / path be `.shortName` itself and be done with it probably.